pipeline/4/Jenkinsfile-tier-0.groovy

```diff
@@ -5,32 +5,6 @@
 def nodeName = "centos-7"
```

Comment:
Can we rename the file to include `RHEL-8` in the filename... like we have for `rhel-7`

Reply:
If we do so, then we will have to change the filename for 5.x job as well. Since both 4.x and 5.x rhel-8 jobs are using the same template.

Reply:
Not making the file name change as discussed